unlock-app/src/config.js

```diff
@@ -71,7 +71,8 @@ export default function configure(
   let requiredNetwork = 'Dev'
   let requiredNetworkId = 1984
   let requiredConfirmations = 12
-  let unlockAddress = ''
+  // Unlock address by default (smart contract deployments yield the same address on a "clean" node as long as long as the migration script runs in the same order)
```

Comment:
I think this is a great step overall, just curious whether there's any possibility of the migration scripts running in a different order and messing things up

Reply:
In dev, yes there is always a risk and I actually think it *will* happen when/if we change the way we deploy the locks but that should be fairly rare.

Reply:
In prod and staging the contract have been deployed and its address will never change.